frontend/src/uploads/UploadContainer.js

```diff
@@ -110,23 +110,38 @@ const UploadContainer = () => {
           if (warnings && checkForWarnings == true) { // ie it is the first attempt to upload (when upload is called from the dialog its set to false)
             setOpenDialog(true)
             setAlertDialogText({
-              title: "Warning: There are similar names in the data to review",
+              title: "Warning: There are errors in the data to review",
               content:(
                 <>
                 <div>
                   <p>
                     Click continue to insert these records as is, or click cancel
                     to exit out and no records will be inserted:
                   </p>
-                  {warnings.map((warningItem, index) => (
-                    <div key={index}>
-                      {Object.keys(warningItem).map(company => (
-                        <div key={company}>
-                          <strong>{company}:</strong> {warningItem[company].join(', ')}
-                        </div>
-                      ))}
-                    </div>
-                  ))}
+                  {
+                    <div>
+                    {warnings.map((warning, index) => (
```

Comment:
Given that we're standardizing what validation functions return, we should update the frontend to handle the standard format (dict of error type -> list of error row numbers).